.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -27,4 +27,4 @@
       <option name="m_ignoreAnnotatedVariables" value="false" />
     </inspection_tool>
   </profile>
-</component>
\ No newline at end of file
+</component>
```

Comment:
Please add the newline back in here, purely a style thing.

Reply:
no idea why that happened. must have been my ide lol